modules/thermo-cycler/thermo-cycler-arduino/lid.cpp

```diff
@@ -17,7 +17,7 @@ void Lid::_i2c_write(byte address, byte value)
   byte error = Wire.endTransmission();
   if (error)
   {
-    Serial.print("Digipot I2C Error: "); Serial.println(error);
+    // Serial.print("Digipot I2C Error: "); Serial.println(error);
```

Comment:
Do we need this?

Reply:
I want to keep it so we know when the i2c isn't working but we don't have anything in the api to catch this message in a good way. The poller in the driver would freeze since there isn't a serial ack at the end of the message. I'm thinking adding an ack after that message would work so that the poller interrupt would catch it..